hoot-rnd/src/main/cpp/hoot/rnd/language/translators/HootServicesTranslatorClient.cpp

```diff
@@ -217,17 +211,21 @@ void HootServicesTranslatorClient::_parseTranslateResponse(
   _translatedText =
     QUrl::fromPercentEncoding(
       QString::fromStdString(replyObj->get<std::string>("translatedText")).toUtf8());
-  LOG_VARD(_translatedText);
```

Comment:
This would crash when the translator service didn't use lang detection and sent nothing back here.